admin/permissions/README.md

```diff
@@ -0,0 +1,39 @@
+## Managing custom roles and permissions
+
+Here's a grab bag of utilities for creating and managing various permissions and roles.
+
+### Set the project and service account
+
+Let's keep our environments consistent by scripting permissions and ACL changes as much as possible.  In general we 
+use service accounts that are unique to each project, so at various points you'll want to change the GCP project
+and the service account when using these scripts.
+
+`GCP_PROJECT` can be either `broad-ddp-dev` , `broad-ddp-test`, `broad-ddp-staging`, or `broad-ddp-prod` 
+
+`SERVICE_ACCOUNT` is the "email" address for the service account, which can be read via `vaultcli read secret/pepper/[dev | test | staging | prod]/v1/conf | jq .gcp.serviceKey.client_email`
```

Comment:
@MocanaAtBroad good idea on vault instructions.  Here they are.

Reply:
vaultcli does not yet exist in ddp-angular repo (probably should).
(and then I don't think you would need jq anyways).
Maybe just shorten `vaultcli` to `vault` ?